vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -43,7 +52,7 @@ ABIViewManager::ABIViewManager(
       m_viewManagerWithChildren{viewManager.try_as<IViewManagerWithChildren>()},
       m_viewManagerWithPointerEvents{viewManager.try_as<IViewManagerWithPointerEvents>()},
       m_viewManagerWithDropViewInstance{viewManager.try_as<IViewManagerWithDropViewInstance>()},
-      m_viewManagerWithOnLayout{viewManager.try_as<IViewManagerWithOnLayout>()} {
```

Comment:
I didn't see `m_viewManagerWithOnLayout` being removed, but it's no longer being set here. Is that a problem?

Reply:
Only use is in this same file:

```cpp
if (m_viewManagerWithOnLayout) {
    m_viewManagerWithOnLayout.OnLayout(viewToUpdate.as<xaml::FrameworkElement>(), left, top, width, height);
```

Reply:
Ouch, good catch.